robowflex_ompl on master does not build [BUG]

<!--- Provide a general summary of the issue in the Title above -->
This is the error I got : 
Errors     << robowflex_ompl:make /home/chamzask/ros/melodic/system/logs/robowflex_ompl/build.make.176.log
In file included from /home/chamzask/ros/melodic/system/src/moveit/moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/ompl_interface.h:41:0,
                 from /home/chamzask/ros/melodic/system/src/robowflex_ompl/include/robowflex_ompl/ompl_interface.h:6,
                 from /home/chamzask/ros/melodic/system/src/robowflex_ompl/src/ompl_interface.cpp:8:
/home/chamzask/ros/melodic/system/src/moveit/moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/constraints_library.h:72:3: error: ‘InterpolationFunction’ does not name a type

## Expected Behavior
<!--- Tell us what should happen -->

## Current Behavior
<!--- Tell us what happens instead of the expected behavior -->

## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->

## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Build robowflex_ompl on master 
2.
3.
4.

## Context (Environment)
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

<!--- Provide a general summary of the issue in the Title above -->

## Detailed Description
<!--- Provide a detailed description of the change or addition you are proposing -->

## Possible Implementation
<!--- Not obligatory, but suggest an idea for implementing addition or change -->

--- a/robowflex_ompl/CMakeLists.txt
+++ b/robowflex_ompl/CMakeLists.txt
@@ -16,7 +16,6 @@ find_package(Boost REQUIRED COMPONENTS serialization)
 find_package(catkin QUIET COMPONENTS
   robowflex_library
   moveit_planners_ompl
-  moveit_ompl_planning_interface
 )
 
 catkin_package(
@@ -24,7 +23,6 @@ catkin_package(
   CATKIN_DEPENDS
     robowflex_library
     moveit_planners_ompl
-    moveit_ompl_planning_interface
   DEPENDS
   INCLUDE_DIRS ${CMAKE_CURRENT_LIST_DIR}/include
   )

